lib/facter/pulpcore_workers.rb

```diff
@@ -0,0 +1,11 @@
+Facter.add(:pulpcore_workers) do
+  setcode do
+    directory = '/etc/systemd/system/multi-user.target.wants'
+    extension = 'pulpcore-worker@*.service'
+    return unless File.exist?(directory)
+
+    Dir[File.join(directory, "*#{extension}")].map do |config|
```

Comment:
Why is the `*` needed? `extension` already contains a glob (and is thus not really well named).
```suggestion
    Dir[File.join(directory, extension)].map do |config|
```